app/controllers/posts_controller.rb

```diff
@@ -5,12 +5,28 @@ class PostsController < ApplicationController
   # GET /posts.json
   def index
     @posts = Post.all
+    @posts_done_homework = current_user.posts.done
+    @posts_pending_homework =  current_user.posts.pending
```

Comment:
@8geonirt @diegosaul402 AFAIK these variables should be part of a separated function, what do you think?